src/telliot_feeds/reporters/reporter_autopay_utils.py

```diff
@@ -154,13 +166,16 @@ async def get_feed_details(self, require_success: bool = False) -> Any:
         # separate items from current feeds
         # create dict of tag and feed_id in current_feeds
         tags_with_feed_ids = {
-            tag: feed_id for tag, feed_id in current_feeds.items() if type(tag) != tuple if len(current_feeds[tag]) > 0
+            tag: feed_id
+            for tag, feed_id in current_feeds.items()
+            if isinstance(tag, tuple)
```

Comment:
using
```python
blah for blah in blah if all(condition1, condition1)
```
would be clearer than this dict comprehension with two if statements..
plz change

Reply:
indeed